doc/installation/mac_install.rst

```diff
@@ -1,95 +1,120 @@
+   .. code-block:: sh
 
- .. code-block:: sh
+      xcode-select --install
 
-        xcode-select --install
+#. Create a conda environment with necessary tools (see also :doc:`conda_tips`)
 
-#.  Install dependencies via Homebrew:
+   .. code:: sh
 
- .. code-block:: sh
+      conda env create -f extras/conda-environment-nest-simulator.yml
 
-       brew install gcc cmake gsl open-mpi libtool
+   .. note::
 
-#.  Create a directory for building and installing NEST (you should always build NEST outside the source code directory; installing NEST in a "place of its own" makes it easy to remove NEST later).
+      To build NEST natively on a Mac with Apple's M1 chip, you need to use Miniforge as 
+      described in :doc:`conda_tips`.
 
-#.  Extract the NEST tarball as a subdirectory in that directory or clone NEST from GitHub into a subdirectory:
+#. Activate the environment with
 
- .. code-block:: sh
+   .. code:: sh
+   
+      conda acvitate nest-simulator
+      
+   This assumes that you have created the environment with its default name ``nest-simulator``.
 
-        mkdir NEST       # directory for all NEST stuff
-        cd NEST
-        tar zxf nest-simulator-x.y.z.tar.gz
-        mkdir bld
-        cd bld
+#. If you want to build NEST with MPI, you must codesign the ``orterun`` and ``orted`` binaries
 
-#.  Configure and build NEST inside the build directory (replacing `gcc-9` and `g++-9` with the GCC  compiler versions you installed with `brew`):
+   a. If you do not yet have a self-signed code-signing certificate, create one as described here:
+      `<https://gcc.gnu.org/onlinedocs/gcc-4.8.1/gnat_ugn_unw/Codesigning-the-Debugger.html>`__
+      (restarting does not appear to be necessary any more).
+   b. Sign your binaries
 
- .. code-block:: sh
+      .. code:: sh
 
-        cmake -DCMAKE_INSTALL_PREFIX:PATH=<nest_install_dir> \
-              -DCMAKE_C_COMPILER=gcc-9 \
-              -DCMAKE_CXX_COMPILER=g++-9 \
-              </path/to/NEST/src>
+         codesign -f -s "gdb-cert" `which orted`
+         codesign -f -s "gdb-cert" `which orterun`
 
- .. code-block:: sh
+      Instead of the ``which`` command you can also give the full path to the binary inside your conda
+      environment.
+      
+      .. note::
+      
+         You will need to sign the binaries every time you update the OpenMPI package in your environment.
 
-        make -j4         # -j4 builds in parallel using 4 processes
-        make install
-        make installcheck
 
-To compile NEST with MPI support, add ``-Dwith-mpi=ON`` as ``cmake`` option.
+Building NEST
+-------------
 
-Troubleshooting
-.................
+1. Download or clone the NEST sources from `<https://github.com/nest/nest-simulator>`__.
 
-If compiling NEST as described above fails with an error message like
+#. Create a build directory outside the NEST sources and move into it.
```

Comment:
```suggestion
#. Create a build directory outside the NEST sources and change into it.
```
The current version contradicts the long-established semantics of *move* when it comes to filesystem operations.